src/apps_plugin.c

```diff
@@ -824,13 +867,15 @@ int read_proc_pid_stat(struct pid_stat *p) {
 
 	// ----------------------------------------
 
-	int set_quotes = (!ff)?1:0;
+	// FIX: set_quotes will always be true!
+	//int set_quotes = (!ff)?1:0;
```

Comment:
Hummmmm... I just notice the 'static' modifier now... sorry again!

On 28-04-2016 13:59, Costa Tsaousis wrote:
>
> In src/apps_plugin.c 
> <https://github.com/firehol/netdata/pull/323#discussion_r61463742>:
>
> > @@ -824,13 +867,15 @@ int read_proc_pid_stat(struct pid_stat *p) {
> >
> >  	// ----------------------------------------
> >
> > -	int set_quotes = (!ff)?1:0;
> > +	// FIX: set_quotes will always be true!
> > +	//int set_quotes = (!ff)?1:0;
>
> no it will not. |ff| is static. The whole idea with these |static 
> profile *ff| is that they maintain their internal structures among 
> calls. They adapt their internal structures to the sizes of the files 
> they see.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/firehol/netdata/pull/323/files/460da7b0b8e59ebcb062b3b1627b452141c1a8ca..21b2e778bcce1d92816265f64175951556e53350#r61463742>
>